Source/NaturalLanguageClassifierV1/Models/ClassifierModel.swift

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright IBM Corporation 2015
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+import Foundation
+import Freddy
+
+extension NaturalLanguageClassifierV1 {
+    
+    public struct ClassifierModel: JSONDecodable {
+        
+        /// Unique identifier for this classifier
+        public let classifierId: String
+        
+        /// Link to the classifer
+        public let url: String
+        
+        /// User-supplied name for the classifier
+        public let name: String?
+        
+        /// The language used for the classifier
+        public let language: String?
+        
+        /// Date and time (UTC) the classifier was created
+        public let created: String?
+        
+        /// Used internally to initialize a `ClassifierModel` from JSON.
+        public init(json: JSON) throws {
+            classifierId = try json.string("classifier_id")
+            url = try json.string("url")
+            name = try json.string("name")
```

Comment:
I think we should be consistent in matching required properties with `try` and optional properties with `try?`.
- `try json.string("name")` throws an error if the `name` key is not present in the JSON. This maps to the idea of a required property, since we produce an error if that property is not found.
- `try? json.string("name")` returns `nil` if the property is not found in the JSON. This maps to the idea of an optional property, since we do not throw an error if the property is missing.

Since `name`, `language`, and `created` are declared as optional, then I think you should decode them with `try?`.

Since the code currently uses `try`, it's interesting that this wasn't caught in a test. Either: